core/src/main/scala/fs2/time/time.scala

```diff
@@ -30,7 +31,7 @@ package object time {
    * @param S           Strategy to run the stream
    * @param scheduler   Scheduler used to schedule tasks
    */
-  def awakeEvery[F[_]](d: FiniteDuration)(implicit F: Async[F], FR: Async.Run[F], S: Strategy, scheduler: Scheduler): Stream[F,FiniteDuration] = {
+  def awakeEvery[F[_]](d: FiniteDuration)(implicit F: Async[F], S: Strategy, scheduler: Scheduler): Stream[F,FiniteDuration] = {
```

Comment:
I think we can do F.start() to run the callback ? Essentially I am nervous about having one strategy bound in Async, and other passed as the implicit. Which one of them will be used?